src/bin/driver.rs

```diff
@@ -292,7 +296,13 @@ async fn run_level(
     client_1: &CollectorClient,
     num_clients: usize,
 ) -> io::Result<()> {
-    let threshold = core::cmp::max(1, (cfg.threshold * (num_clients as f64)) as u64);
+    let threshold = if let Mode::WeightedHeavyHitters { threshold } = cfg.mode {
```

Comment:
I only see `attribute_based_metrics` in the config.json, no WHH

Reply:
Right, the "mode of operation" would be one or the other. See `enum Mode`.